src/foreman.ts

```diff
@@ -84,6 +84,10 @@ async function authenticate(token: string): Promise<void> {
   await exec("foreman", ["github-auth", token]);
 }
 
+async function addArtifactoryToken(url: string, token: string): Promise<void> {
+  await exec("foreman", ["artifactory-auth", url, token]);
```

Comment:
Oh, do we need to check the version before we can do this? Or can we just count on the error carrying through and hope that folks can figure it out?

Reply:
hmmm, we should probably warn if the version selected is below the supported version